prow/plank/README.md

```diff
@@ -0,0 +1,41 @@
+# Plank
+
+Plank is the controller that manages the job execution and lifecycle for jobs running in k8s.
+
+### Usage
+```bash
+$ bazel run //prow/cmd/plank -- --help
+```
+
+### Configuration
+The following job template is based on the format for `spyglass`.
+
+```yaml
+# config.yaml
+
+plank:
+  allow_cancellations: true
+  job_url_prefix: 'https://<domain>/view/gcs/<bucket-name>'
+  job_url_template: 'https://<domain>/view/gcs/<bucket-name>/pr-logs/pull/{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}'
+  report_template: '[Full PR test history](https://<domain>/pr-history?org={{.Spec.Refs.Org}}&repo={{.Spec.Refs.Repo}}&pr={{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}})'
+  default_decoration_config:
+    timeout: 14400000000000 # 4h in nano seconds
+    grace_period: 15000000000 # 15s in nano seconds
+    utility_images:
+      clonerefs: gcr.io/k8s-prow/clonerefs:v20190221-d14461a
+      initupload: gcr.io/k8s-prow/initupload:v20190221-d14461a
+      entrypoint: gcr.io/k8s-prow/entrypoint:v20190221-d14461a
+      sidecar: gcr.io/k8s-prow/sidecar:v20190221-d14461a
+    gcs_configuration:
+      bucket: <bucket-name>
+      path_strategy: explicit # or `legacy`, `single`
+      default_org: <github-org> # should not need this if `strategy` is set to explicit
+      default_repo: <github-repo># should not need this if `strategy` is set to explicit
+    gcs_credentials_secret: <secret-name> # the name of the secret that store the GCP service account credential JSON file
```

Comment:
This would be a good place (or somewhere in the doc) to mention we expect the key to be `service-account.json`

Reply:
there's a note right below this code block stating that. do you prefer me to put it in the code sample?

Reply:
Wow, I totally missed that. Yeah, can we put it up here?